Manual/manJHUGenerator.tex

```diff
@@ -1410,13 +1453,42 @@ \section{Modified Zff couplings in MCFM}
     cd bin 
     ./mcfm
 \end{verbatim}
+The SM Zff couplings in MCFM-JHUGen are defined as:
+
+\begin{eqnarray*}
+  \verb|daz_(*)_left| = \frac{1}{\sin\theta_{W}\cos\theta_{W}} \left(T^{3}_{f}-Q_{f} \sin^2 \theta_{W}\right) \\
+  \verb|daz_(*)_right| = -\frac{1}{\sin\theta_{W}\cos\theta_{W}} Q_{f} \sin^2 \theta_{W}
+\end{eqnarray*}
+where $T^{3}_{f}$ and $Q_{f}$ are the weak isospin and charge of the fermion in consideration and $\theta_{W}$ is the weinberg angle.
+
+We also support modified Zff couplings in MCFM-JHUGen matrix elements for certain MELA Processes. As in the JHUGen case, all of the Zff couplings in decay 
+are set directly in the interface. We only consider anomalous Zqq couplings in production as shifts to the SM values for these couplings. If any anomalous Zff coupling is set to a non-zero value in MELA all of the the other Zff couplings will be assumed to be 0
+unless they are also explicitly set to some non-zero value. An example for setting Zff couplings in MCFM-JHUGen matrix elements with MELA is given below:
+
+\begin{verbatim}
+mela->setProcess(TVar.bkgZZ_SMHiggs, TVar.MCFM, TVar.ZZGG)
+mela->selfDAZffcoupl[gAZff_ZllRH][0] = 0.46238
+mela->selfDAZffcoupl[gAZff_ZllLH][0] = -0.53762
+mela->selfDAZffcoupl[gAZff_ZuuRH] = -0.30825
+mela->selfDAZffcoupl[gAZff_ZddRH] = 0.15413
+mela->selzDAZffcoupl[gAZff_ZnnRH]b = 1
+mela->selfDAZffcoupl[gAZff_uZLH] = 0.1 
+\end{verbatim}
+
+This would correspond to SM Zll couplings, and SM Z couplings to right handed quarks. The Z coupling to right-handed neutrinos is set to 1 and the SM Z coupling to left-handed
+up-type quarks in production is shifted by 0.1. Again in this example, since the other Zff couplings are not set, they are set to 0 in the matrix element calculation.
 
 \section{Release notes}
 \noindent 
-In going from \verb|v7.5.2| to \verb|v7.5.3| (June 28, 2023), the updates are as follows:
+In going from \verb|v7.5.2| to \verb|v7.5.3| (August 25, 2023) the updates are as follows:
 \noindent
 \begin{itemize}
-\item Added width scheme 4 (running) in JHUGen and MCFM-JHUGen processes
+\item Add anomalous Zff couplings in off-shell $gg\to H \to 4f$ and $gg \to 4f$ MCFM matrix elements (MCFM Process 128,131,132).
+\item Add anomalous Zff couplings for on-shell Higgs decays in JHUGen matrix elements
+\item Add interface for anomalous Zff couplings in MELA
+\item Add width scheme 4 (running) in JHUGen and MCFM-JHUGen processes
+\item Improve numerical stability in off-shell $gg \to 4f$ processes at low $q^2$ 
+\item Small fix to JHUGenLexicon Warsaw to Higgs basis translation without custodial symmetry
```

Comment:
Please add more details on which coupling is fixed.

Reply:
I think that would be too much to include in a single line. Historically we do not have any detailed explanation of what the bugs are in the release notes. Unless you specifically want me to just write the "Small fix to c_z{\bx} in JHUGenLexicon Warsaw to Higgs basis translation without custodial symmetry"

Reply:
"Fix to c_z{\bx} in JHUGenLexicon Warsaw to Higgs basis translation without custodial symmetry" works, thanks!

Reply:
@jdavis36 Did you get the chance to also implement this change? I think this short note on which coupling is fixed should be added before the PR is approved.